src/org-unit-tree/OrgUnitTree.component.js

```diff
@@ -42,7 +41,7 @@ class OrgUnitTree extends React.Component {
                 : undefined,
             loading: false,
         };
-        if (props.root.children instanceof ModelCollection) {
+        if (!props.root.children instanceof Model) {
```

Comment:
Yup, absolutely. I have run the code once and ran tests but missed this some how. My intention was to check if this is not of type Model. But even that could go wrong when we get a d2 Model object whose children has some object not being a model collection. My initial check was checking if toArray exists on the children but I felt it not being a good fix. Shall I go with the above check then with one more condition on top to check if children exists in the first place. 
if (props.root.children && !Array.isArray(props.root.children.toArray) && typeof props.root.children.toArray === 'function')